### DISCUSSION TITLE
no top/bottom/left/right-{number} feature?

### DISCUSSION THREAD

**Discussion by kertahayu** [tailwindlabs/tailwindcss#2628] [Help] [answered]
Why is the `top-{number}` `left-{number}` feature only available on tailwindui?

**Comment by JNavith** [comment, marked as answer]
Currently, you can enable the experimental `extendedSpacingScale` feature
https://github.com/tailwindlabs/tailwindcss/releases/tag/v1.7.0#extended-spacing-scale

However, Tailwind 2.0 will remove this flag, and per @adamwathan's reply below https://github.com/tailwindlabs/tailwindcss/discussions/2628#discussioncomment-106914, it seems like a future commit will also remove the extended `inset` scale. So, follow the instructions in https://github.com/tailwindlabs/tailwindcss/discussions/2628#discussioncomment-106912 instead.

**Reply by adamwathan** [reply]
Quick note, the 2.0 build is going to be a bit more restricted than what was in that experimental spacing scale. The build was way too big and causing slow downs in dev tools.

**Reply by kertahayu** [reply]
OK, meanwhile, I'll be configuring the inset myself to prevent future errors. and think about your suggestions using margins "The build was way too big and causing slow downs in dev tools".

thanks for the answer. this is a great tool.

**Comment by adamwathan** [comment]
It's available in Tailwind, you just need to add the values you need. We ship with just `0` and `auto` by default to keep the build a bit smaller since usually you can accomplish what you need by combining `top-0` and `mt-16` sort of thing.
